game/hud/src/components/HUD/index.tsx

```diff
@@ -61,12 +61,20 @@ class HUD extends React.Component<HUDProps, HUDState> {
         }]
     }
   }
-  
+
   componentWillMount() {
+    window.removeEventListener("resize", this.handleResize);
   }
 
   componentDidMount() {
     this.props.dispatch(initializeHub());
+    window.addEventListener("resize", this.handleResize);
```

Comment:
No magic:  https://developer.mozilla.org/en-US/docs/Web/Events/resize

Leave as is.